app/javascript/packs/pages/binx_admin.js

```diff
@@ -2,6 +2,7 @@ import log from "../utils/log";
 import moment from "moment-timezone";
 import LoadFancySelects from "../utils/LoadFancySelects";
 import BinxAdminInvoices from "./binx_admin_invoices.js";
+import BinxGraphs from "./binx_graphs.js"
```

Comment:
I think we're getting enough admin js that we should create an `admin` folder in the pages folder, and put all the admin stuff in there.

Reply:
My thoughts on naming conventions for this: 
- All classes/functions should be prefixed with `BinxAdmin{{Name}}`
- The filenames should be just the final part

e.g. for this, filename would be `graphs.js` and the classname would be `BinxAdminGraphs`